opl/investigator/config.py

```diff
@@ -75,6 +75,10 @@ def load_config(conf, fp):
         assert not conf.history_es_server.endswith("/")
         conf.history_es_index = data["history"]["es_index"]
         conf.history_es_query = data["history"]["es_query"]
+        if "es_server_user" in data["history"]:
```

Comment:
Could you please align `es_server_user` variable name (and others) with `history_es_index` (i.e. adding `history_` prefix)?